### PR TITLE
Use clang-tidy to get remaining snakes in ride/coaster

### PR DESCRIPTION
**Pull request by duncanspumpkin**
This was automated so will need a little extra bit of review just to make sure its not done anything stupid.

### CODE REVIEW

File: src/openrct2/ride/coaster/AirPoweredVerticalCoaster.cpp [hunk 1]
```diff
@@ -164,7 +164,7 @@ enum
     SPR_AIR_POWERED_VERTICAL_RC_BANKED_QUARTER_TURN_5_FRONT_SE_NE_PART_0 = 22333,
 };
 
-static ImageId air_powered_vertical_rc_get_support_colour(PaintSession& session)
+static ImageId AirPoweredVerticalRcGetSupportColour(PaintSession& session)
```
Comment (Gymnasiast) [inline src/openrct2/ride/coaster/AirPoweredVerticalCoaster.cpp:167]: RC

Probably do a find and replace for this.

**Comment by Gymnasiast** [comment]
This is changing 6000 lines. That's too much if we also have to check the diff for any cock-ups by clang-tidy.

**Review by Broxzier — COMMENTED** [review]
I've verified that there are no clang-tidy mistakes in this PR by taking all removed and all added lines from the diff, made them all lowercase and removed all underscores, then compared side-by-side. The only differences that then showed up were white-space changes by clang-format, and the tables in SplashBoats.cpp.

**Review by Gymnasiast — APPROVED** [review]
Assuming that Broxzier’s method was sufficient, this is good to go.

**Pull request merged**